CHANGELOG.md

```diff
@@ -1,656 +1,580 @@
-## puppetlabs-puppetdb changelog
+<!-- markdownlint-disable MD024 -->
```

Comment:
Do we have to rewrite the whole changelog? I think this project uses not the github changelog generator? That allows to put the old changelog in a HISTORY.md and keep it. Makes the diff way more readable.

Reply:
fixed

Reply:
Re-opening, see #399

Reply:
Oh crap, the changelog is not generated with the `rake changelog` task :unamused:.

I recall triaging all issues in various modules, so I guess we want to do this too for consistency?  The current tooling will then generate a *full* changelog without HISTORY.md on each release, but as the HISTORY.md contain the old changelog, maybe it will not be too bad?

Reply:
I labelled issues, and regenerated CHANGELOG.md by hand (because it is not currently in the main branch so the GitHub action does not think it must be updated).  This is a PITA only for this release as the next ones will not have this corner case.